src/OpenLoco/OpenLoco.cpp

```diff
@@ -548,6 +549,15 @@ namespace OpenLoco
         }
     }
 
+    // This is called when the game requested to end the current tick early.
+    // This can be caused by loading a new save game or exceptions.
+    static void tickInterrupted()
+    {
+        EntityTweener::get().reset();
+        std::cout << "tick interrupted" << std::endl;
+        return;
```

Comment:
The return statement seems a bit superfluous. Also, this is a good time to switch to using `Console::log`:
```suggestion
        EntityTweener::get().reset();
        Console::log("Tick interrupted");
```